content/01-contributing-oss/slides/index.en.md

```diff
@@ -0,0 +1,162 @@
+---
+outputs:
+- Reveal
+title: Introductory slides
+hidden: true
+layout: list
+weight: 1
+output: hugodown::md_document
+rmd_hash: f0da839050cdbe81
+
+---
+
+Contributing to Open Source Software
+====================================
+
+:wave: Welcome!
+
+------------------------------------------------------------------------
+
+I'm Bob, I like science, people, code and records.
+
+------------------------------------------------------------------------
+
+<img src="/images/hackfest2020.PNG" height=200/>
+
+- A one month celebration of Open Source on GitHub.
+
+- Meaningful contributions to the open source ecosystem.
+
+- *4* pull requests to a public repository on GitHub to get a t-shirt or a tree (don't need to be accepted).
```

Comment:
but a PR does now need to be approved by the repo maintainer to count (this is new).

Reply:
Is this so - I thought it would be rejected if marked as spam, but does not need approval to count?
![image](https://user-images.githubusercontent.com/34244196/95070296-d1ba3280-06ff-11eb-9617-956993c53143.png)
Although my profile looks different!
![image](https://user-images.githubusercontent.com/34244196/95070363-e8f92000-06ff-11eb-93ae-fc7721b21914.png)
I'm confused! Is there something definitive you've found somewhere, please?

Reply:
This is a VERY recent development https://hacktoberfest.digitalocean.com/hacktoberfest-update
As it says, PR's will only count if accepted by one of those 3 methods listed.

Reply:
yeah i thought it was in the digital ocean blog i read this morning. I'm talking about un-accepted PRs (accepted PRs in a hacktoberfest tagged repo automatically count, I think).

Reply:
Ta!

Reply:
I've linked to the new rules from the talk.

Reply:
I believe PRs opened prior to the changes being made by hacktoberfest will count without having to follow the new rules (hence your two exisitng PRs are `In Review` rather than `Pending`